trunk/modules/python/dataset/ingest/labeled.py

```diff
@@ -1,6 +1,13 @@
 import os
 import numpy as np
 
+def mostCommon(stuff, func):
+    '''Find the most common result of func yielded by stuff'''
+    from collections import Counter
+    counter = Counter([func(s) for s in stuff])

```

Comment:
Can you test the speed of this against statistics.mode() and/or scipy.stats.mode()? This is already a long running piece of code, so we should optimize it for our large datasets.